packages/models/test/huggingface/streaming-text-generation-response.txt

```diff
@@ -0,0 +1,40 @@
+data:{"token":{"id":304,"text":" To","logprob":-0.012886047,"special":false},"generated_text":null,"details":null}
```

Comment:
Up to you, but I would be fine with only keeping one of these files (the one with the stop token) as that seems good enough?

Reply:
I like having both since they have different behaviors and I'm testing both.